avm/res/compute/virtual-machine/modules/nic-configuration.bicep

```
@@ -31,7 +31,7 @@ import { roleAssignmentType } from 'br/public:avm/utl/types/avm-common-types:0.5
 param roleAssignments roleAssignmentType[]?
 
 module networkInterface_publicIPAddresses 'br/public:avm/res/network/public-ip-address:0.6.0' = [
-  for (ipConfiguration, index) in ipConfigurations: if (contains(ipConfiguration, 'pipConfiguration') && !contains(
+  for (ipConfiguration, index) in ipConfigurations: if (contains(ipConfiguration, 'pipConfiguration') && !empty(ipConfiguration.pipConfiguration) && !contains(
     ipConfiguration.pipConfiguration,
     'publicIPAddressResourceId'
   )) {
```

```suggestion
  for (ipConfiguration, index) in ipConfigurations: !empty(ipConfiguration.?pipConfiguration) && empty(    ipConfiguration.?pipConfiguration.?publicIPAddressResourceId) {
```
Should work - though Bicep would probably change the formatting & indentation if opened locally